### PR TITLE
Created tabs for Notifications Page

### PR DESCRIPTION
**Pull request by SoumyadiptoPal**
solved Issue #55
Created the notifications page.
Used a hard-coded object array for development purpose. Each object contains a header, message, time and type.  Also added 3 images for critical, warning and info types of messages.
screen (1)

### CODE REVIEW

File: screens/Notifications.tsx [hunk 1]
```diff
@@ -1,10 +1,108 @@
...
+  const pressedWarning=()=>{
+    setOption("Warning");
+    const filteredArray=filterArrayByField("Warning");
+    setNotificationArray(filteredArray);
+  }
+  const pressedInfo=()=>{
+    setOption("Info");
+    const filteredArray=filterArrayByField("Info");
+    setNotificationArray(filteredArray);
+  }
```
Comment (mdshamoon) [inline screens/Notifications.tsx:66]:
```suggestion
const handleOptionPress = (option) => {
setOption(option);
if (option === "All") {
setNotificationArray(Notifications);
} else {
const filteredArray = filterArrayByField(option);
setNotificationArray(filteredArray);
}
};
```

File: screens/Notifications.tsx [hunk 2]
```diff
@@ -1,10 +1,108 @@
...
+          <Text style={[styles.text, (option==="Warning")?styles.active:{}]}>WARNING</Text>
+        </TouchableOpacity>
+        </View>
+        <View style={(option==="Info")?styles.option:{}}>
+        <TouchableOpacity onPress={pressedInfo}>
+          <Text style={[styles.text, (option==="Info")?styles.active:{}]}>INFO</Text>
+        </TouchableOpacity>
+        </View>
+      </View>
+      <ScrollView>
```
Comment (mdshamoon) [inline screens/Notifications.tsx:98]:
```suggestion
    const RenderOptionButton = ({ label, selectedOption, onPress }) => {
  const isActive = selectedOption === label;
  return (
    <View style={isActive ? styles.option : {}}>
      <TouchableOpacity onPress={onPress}>
        <Text style={[styles.text, isActive ? styles.active : {}]}>
          {label}
        </Text>
      </TouchableOpacity>
    </View>
  );
};

// Usage:
<View style={styles.navBar}>
  <RenderOptionButton label="All" selectedOption={option} onPress={pressedAll} />
  <RenderOptionButton label="Critical" selectedOption={option} onPress={pressedCritical} />
  <RenderOptionButton label="Warning" selectedOption={option} onPress={pressedWarning} />
  <RenderOptionButton label="Info" selectedOption={option} onPress={pressedInfo} />
</View>
```

**Review by mdshamoon — CHANGES_REQUESTED** [review]
Please format the code with prettier. and use svg instead of png for images

**Comment by ajaman190** [comment]
> Please format the code with prettier. and use svg instead of png for images

Unfortunately, we cannot use SVG directly in a React Native project. However, I have come across two better solutions:
1. Use the `react-native-svg-transformer` package to render SVG code.
2. If the icon is of a single color, convert the icons into a font family and use it as a normal font. This approach also helps reduce the overall app size.

**Comment by mdshamoon** [comment]
Closing in favor of #62

**Pull request closed without merging**